crates/starknet-devnet/src/cli.rs

```diff
@@ -135,7 +135,9 @@ pub(crate) struct Args {
     #[arg(env = "CHAIN_ID")]
     #[arg(value_name = "CHAIN_ID")]
     #[arg(default_value = "TESTNET")]
-    #[arg(help = "Specify the chain ID;")]
+    #[arg(help = "Specify the chain ID. Possible values are:
+- \"MAINNET\", \"TESTNET\" - predefined chain IDs
+- <USER_SUPPLIED> - custom chain ID, defined by user. Have to contain only ASCII characters.")]
```

Comment:
What if `--chain-id` is specified together with `--fork-network`? I suggest we make them conflict with each other by relying on https://docs.rs/clap/latest/clap/struct.Arg.html#method.conflicts_with

Reply:
It will be overridden by fork-network

Reply:
Instead of silently overriding, I think it's better to inform the user, preferably by means of erroring.

Reply:
[8c683f](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/549/commits/8c683f99edac5b929895f6f711925c2cf84056c4)